Makefile

```diff
@@ -111,6 +105,7 @@ else
 	scBuildImageTarget = .scBuildImage
 endif
 
+# Even though we migrated to dep, it doesn't replace the `glide nv` command
 NON_VENDOR_DIRS = $(shell $(DOCKER_CMD) glide nv)
```

Comment:
I think some go tooling by default knows not to look at the `vendor` directory. Need to look at what this was used for.

Reply:
If you run `go test ./...` using 1.9+, it won't run the tests in vendor. I believe all go tools (not just test but say vet and fmt) are also well behaved.

Reply:
Okay. The only place we use this output is as input to `go vet`. We can investigate this later.